IGI.Lab3/CarPark.WebAPI/CarPark.WebAPI.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.4" />
```

Comment:
I guess we do not need authentication here.

Reply:
I thought it was right, because in most projects this will be necessary.
Plus, I remembered what I was taught in internal courses, and this is a good exercise

Reply:
"PS: Any User role usage?"
At the moment, the application uses the role of Admin and Just registered user in the system

Reply:
Please, make a task according to the description in the future)

Reply:
PS The authorization in API should be in the 6th lab that was excluded now)